.travis.yml

```diff
@@ -20,10 +19,11 @@ addons:
    postgresql: "9.4"
 
 before_script:
-   - psql -c "CREATE ROLE full_access;" -U postgres
+   # Used by (so far non-existent) spec tests for xmlforms
```

Comment:
It gets created in the `initialistPosgres` function which gets called before any tests run: https://github.com/medic/medic-analytics/blob/2124.reflect-edited-and-deleted-rebased/spec/couch2pg/index.js#L58-69

I don't really want to do any scripting in the travis file if I can avoid it, because the travis file doesn't get used locally, and you should be able to run all tests locally.